scripts/release/update-apple-device-repo.js

```diff
@@ -10,9 +10,9 @@ const commit = process.env.BSK_SHA
 function updateAppleDeviceRepo (platform = 'ios') {
     console.log(`running updateAppleDeviceRepo for ${platform}`)
 
-    const projectFilePath = filepath(`../${platform}/DuckDuckGo.xcodeproj/project.pbxproj`)
+    const projectFilePath = filepath(`../../${platform}/DuckDuckGo.xcodeproj/project.pbxproj`)
     const bskPackageRegex = new RegExp(
-        /(repositoryURL = "https:\/\/github\.com\/duckduckgo\/BrowserServicesKit";\s+requirement = {\s+kind = )(exactVersion)(;\s+)(version = 21\.0\.0;)/
+        /(repositoryURL = "https:\/\/github\.com\/duckduckgo\/BrowserServicesKit";\s+requirement = {\s+kind = )(exactVersion)(;\s+)(version = \d+.\d+.\d+;)/
```

Comment:
Could you extract this regex to an exported variable, and then add a couple of test-cases for it? it would have a nice side-effect of documented what you expect